client/src/main/java/io/pravega/client/stream/EventWriterConfig.java

```diff
@@ -27,25 +27,12 @@
      * The maximum amount of time, in milliseconds, which a transaction can run before it is considered failed.
      */
     private final long transactionTimeoutTime;
-    /**
-     * The maximum amount of time, in milliseconds after a scale operation has been initiated before a transaction is timed out. 
-     */
-    private final long transactionTimeoutScaleGracePeriod;
-    
+
     public static final class EventWriterConfigBuilder {
         private int initalBackoffMillis = 1;
         private int maxBackoffMillis = 20000;
         private int retryAttempts = 10;
         private int backoffMultiple = 10;
         private long transactionTimeoutTime = 30 * 1000 - 1;
-        private long transactionTimeoutScaleGracePeriod = -1;
```

Comment:
I'm not onboard with the idea of removing the parameter altogether as this is a breaking API change and it will break things like connectors. Could we deprecate the parameter (it will do nothing, but it is fine), announce it in the 0.3.0 release, and remove it altogether in future releases?

My suggestion is to annotate it:

https://docs.oracle.com/javase/8/docs/api/java/lang/Deprecated.html

and explain as part of the javadoc of the parameter what is going on. What do you think?

Reply:
done